src/styles/index.scss

```diff
@@ -1,3 +1,66 @@
 body {
   margin: 0;
 }
+nav a {
```

Comment:
Ensure that this SCSS file is compiled to CSS, as browsers do not natively support SCSS. This is crucial for the styles to be applied correctly in the HTML file.